DragonFruit.Link.Tests/Game/SteamGlobalGameStatsTests.cs

```diff
@@ -14,7 +14,7 @@ public class SteamGlobalGameStatsTests : SteamApiTest
         [TestMethod]
         public void GetSteamGlobalGameStatsTest()
         {
-            var metric = new List<string>() { "global.map.emp_isle" };
+            var metric = new List<string> { "global.map.emp_isle" };
```

Comment:
can be simplified further
```suggestion
            var metric = new[] { "global.map.emp_isle" };
```

Reply:
A little late in seeing this. I used a list here since it's easier to work with lists. But just for my curiosity, are using arrays better than lists?

Reply:
depends on what you're doing. fixed size, yes

a list is an array-backed class that scales up when you add more items than there are available 'slots'

Reply:
Right, so in this case creating a new array of something I know won't change its size makes more sense than using a list.